quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/integration/resteasy/ResteasyVertxProvider.java

```diff
@@ -20,16 +20,24 @@
 import io.quarkus.arc.Arc;
 import io.quarkus.vertx.http.runtime.CurrentVertxRequest;
 import io.vertx.ext.web.RoutingContext;
-import org.jboss.resteasy.core.ResteasyContext;
 import org.keycloak.common.util.ResteasyProvider;
 
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Optional;
 
 public class ResteasyVertxProvider implements ResteasyProvider {
 
+    private static final ThreadLocal<Map<Class<?>, Object>> contextualData = new ThreadLocal<Map<Class<?>, Object>>() {
+        @Override
+        protected Map<Class<?>, Object> initialValue() {
+            return new HashMap<>();
+        };
+    };
+
```

Comment:
I see that's the almost identical impl as provided by ResteasyContext, instead of the ThreadLocalStack, but it's not necessary. LGTM